libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -379,9 +379,13 @@ bool AP_Mission::verify_command(const Mission_Command& cmd)
     case MAV_CMD_DO_DIGICAM_CONFIGURE:
     case MAV_CMD_DO_DIGICAM_CONTROL:
     case MAV_CMD_DO_SET_CAM_TRIGG_DIST:
+#if defined(HAL_PARACHUTE_ENABLED) && HAL_PARACHUTE_ENABLED
```

Comment:
It is an error if these defines are not, actually, defined.
```suggestion
#if HAL_PARACHUTE_ENABLED
```

similarly elsewhere

Reply:
../../libraries/AP_Mission/AP_Mission.cpp:382:5: error: "HAL_PARACHUTE_ENABLED" is not defined, evaluates to 0 [-Werror=undef]
  382 | #if HAL_PARACHUTE_ENABLED
      |     ^~~~~~~~~~~~~~~~~~~~~
compilation terminated due to -Wfatal-errors.
cc1plus: some warnings being treated as errors

Reply:
> In FMUV3, HAL_PARACHUTE_ENABLED is not defined. Your comment is incorrect.
> 
> ../../libraries/AP_Mission/AP_Mission.cpp:382:5: error: "HAL_PARACHUTE_ENABLED" is not defined, evaluates to 0 [-Werror=undef] 382 | #if HAL_PARACHUTE_ENABLED | ^~~~~~~~~~~~~~~~~~~~~ compilation terminated due to -Wfatal-errors. cc1plus: some warnings being treated as errors

```
#include <AP_Parachute/AP_Parachute_config.h>
```

Reply:
@peterbarker san
I have incorporated the review comments.